import-service/handlers/importProductsFile.ts

```diff
@@ -0,0 +1,38 @@
+import { APIGatewayProxyHandler, APIGatewayProxyEvent } from 'aws-lambda';
+import { RESPONSE } from 'lib/constants/response';
+
+import { importService } from 'lib/services/import';
+
+import 'source-map-support/register';
+
+export const importProductsFile = async (event: APIGatewayProxyEvent) => {
+  console.info('Executing lambda', event);
+
+  try {
+    const { name } = event.queryStringParameters;
+
+    if (!name) {
```

Comment:
It would be nice to validate the file format as well as your parser function is working only with .csv files.